mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -49,6 +50,34 @@ export class CompareRunView extends Component {
     Utils.updatePageTitle(pageTitle);
   }
 
+  adjustTableColumnWidth(numRuns) {
```

Comment:
Why need this handler:

Previously the parameters / metrics table block, all columns are aligned exactly, and when page width changed, the columns will auto change size.

But, after add the vertical scroll bar to parameters/metrics block, things changed, the columns in the 2 blocks they are not aligned, and it cannot respond to the page size changing. Now we need additional js to do the work, i.e. this function.

Reply:
https://user-images.githubusercontent.com/19235986/151131306-af5923ad-46f8-4f37-805c-e25c3827a925.mov